group_vars/all/vars.yml

```diff
@@ -78,6 +80,7 @@ library_github_keys:
   - https://github.com/escowles.keys
   - https://github.com/hackartisan.keys
   - https://github.com/hectorcorrea.keys
+  - https://github.com/ishasinha1.keys
```

Comment:
This list gives access to the `pulsys` user, and you will just need access to the `deploy` user, so I don't think you will need to be on this list either.